plugin/pkg/scheduler/algorithm/predicates/predicates_test.go

```diff
@@ -4886,3 +4886,100 @@ func TestInterPodAffinityAnnotationsWithMultipleNodes(t *testing.T) {
 		}
 	}
 }
+
+func benchmarkPodMatchesNodeLabels(pods []*v1.Pod, nodes []*v1.Node, b *testing.B) {
```

Comment:
The original purpose of those benchmark is to run `pprof` to identify which funcs consume most time (#47319). Had a offline discussion with @ravisantoshgudimetla , that'll be better to include such info into sched_perf although no solution for now; if can not add into sched_perf, I'll find a way to measure that as you said.